easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-21.14.eb

```
@@ -30,11 +30,11 @@ docurls = [
 
 toolchain = SYSTEM
 
-# Download and license at https://chemaxon.com/products/marvin
-# Once logged in download files can be found here:
-# https://chemaxon.com/products/marvin/download#download
 sources = ['marvin_linux_%(version)s.rpm']
 checksums = ['3102f30479a365758fd14ca5efcdacfe0624db80a35b146ba9a5d07e948bd8dc']
+download_instructions = f"""{name} requires license and manual download from https://chemaxon.com/products/marvin
```

since you did so for many others
```suggestion
download_instructions = f"""{name} requires license and manual download from {homepage}
```
any reason not to?

---

3e9cad8